.github/workflows/windows.yml

```diff
@@ -49,7 +49,7 @@ jobs:
         shell: powershell
         run: |
           conda activate autodiff
-          cmake -S . -B .build
+          cmake -S . -B .build -DAUTODIFF_TEST_SANITIZE=ON
```

Comment:
I don't think this works in Windows. I'll remove this (the only thing missing to get CI fully passing)